en/docs/sdks/integrate-a-js-app.md

```diff
@@ -0,0 +1,43 @@
+# Integrate a JavaScript app
+Follow the steps given below to authenticate users to a sample JavaScript service provider with OpenID Connect.
+
+## Prerequisites
+- Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
+
+## Create a service provider
+
+{!fragments/register-a-service-provider.md!}
```

Comment:
We need a way to incorporate the SP configs relevant to the sample we are using.